packages/server/src/fhir/operations/README.md

```diff
@@ -58,13 +58,13 @@ interface ProjectInitParameters {
   ownerEmail?: string;
 }
 
-export async function projectInitHandler(req: Request, res: Response): Promise<void> {
+export async function projectInitHandler(req: FhirRequest): Promise<FhirResponse> {
```

Comment:
Thanks for updating the docs as well!

Reply:
const operation = getOperationDefinition('Measure', 'evaluate-measure');

Reply:
"Replace in files" found it 🤣